client/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+
+class Checkbox extends Component {
+  state = {
+    isChecked: false,
+  }
+
+  toggleCheckboxChange = () => {
+    const { handleCheckboxChange, label } = this.props;
+
+    this.setState(({ isChecked }) => (
```

Comment:
this is code from task 2, please don't mix it up

Reply:
ok i will recommit it. thanks.